client/homebrew/brewRenderer/brewRenderer.jsx

```diff
@@ -173,6 +174,27 @@ const BrewRenderer = (props)=>{
 		}));
 	};
 
+	const handleStyle = (newStyle)=>{
```

Comment:
This step seems way too complex. We should be able to reuse a similar process to what we did with setting the zoom. I.e.:

1) User clicks a button in the toolbar
2) Handler function from the parent gets called with the new setting
3) Renderer in the parent uses the new setting to conditionally change its style

If we need a whole custom processor for styles like this something has gone wrong